main.go

```diff
@@ -74,7 +106,10 @@ func readFlags() (params AppConfigParams) {
 	return
 }
 
-func getConfig(params AppConfigParams) ([]string, error) {
+var n int
```

Comment:
Did you mean to include this `n` variable?

Reply:
No. Thanks for spotting it.